app/controllers/user_controller.rb

```diff
@@ -0,0 +1,78 @@
+class UserController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+  before_action :autehnticate_user!
```

Comment:
Typo here. Also, is this referring to a method somewhere or is this Rails-y magic?